client/src/pages/UserProfile.js

```diff
@@ -124,19 +120,19 @@ const useStyles = makeStyles(theme => ({
   cuisineText: {
     color: 'white',
     fontStyle: 'bold',
-  },
-  // Footer
-  mapImage: {
-    width: "100%",
-  },
+  }
 }));
 
 export default function ProfilePage(props) {
   const { user } = useContext(UserContext);
-  const map =
-    'https://lun-us.icons8.com/a/ybbxUKFceUicCgkzopwXcA/njrM31xG9kms8VRKbon19A/Slice.png';
   const classes = useStyles();
-
+  const location = {
+    address: '2369 wildwood cres., Pickering, ON',
+    lat: 43.85537,
+    lng: -79.08654899999999,
+    radius: 5,
```

Comment:
Should this still be hardcoded?

Reply:
the dynamic data is in another branch which is under review still